pkg/controller/stacks/install/installjob.go

```diff
@@ -72,6 +72,12 @@ func createInstallJob(i v1alpha1.StackInstaller, executorInfo *stacks.ExecutorIn
 		namespace = o.Namespace
 	}
 
+	img, err := i.GetSpec().ImageWithSource(i.GetSpec().Package)
```

Comment:
here's a place where it's not clear if the caller of `i.GetSpec()` should be checking for nil before proceeding.  Knowing how the code is implemented it shouldn't ever be nil but the contract isn't clear just by looking at the interface. Does this `i.GetSpec()` need to return a pointer?

also, you can probably just call `i.GetSpec()` once instead of twice on this line and another time on 78 below